src/main/java/sc/iview/commands/demo/advanced/ShowSegmentationDemo.java

```diff
@@ -121,6 +121,8 @@ public void run() {
 
             // Make the segmentation mesh a child of the parent
             v.addChild(isoSurfaceMesh);
+            // We need to publish our node to sciview to update the UI
+            sciView.publishNode(isoSurfaceMesh);
```

Comment:
So the current pattern is:
```kotlin
var v = sciview.addVolume(...)
v.addChild(mesh)
sciview.publishNode(mesh)
```
?
Is there a way to make this happen automatically, by rolling the publication into one of the other API calls?

If the idea is to "build up" the volume's structure before publishing the finished product, maybe we could use a builder pattern, or some kotlinesque/functional thing?
```kotlin
var v = sciview.addVolume(...).withConfig { it.addChild(mesh) }
```
Such that publication happens when the `withConfig` finishes? There are surely a few different ways to pet this particular cat, just wondering if we can reduce the chance of forgetting to publish after setting things up.

Reply:
I'm open to this, but I'm not sure I'm up for it for `1.0.0`. This sounds like adding a kotlin extension to `Node`. 

Issues like:
Node's actually don't store information about whether they were published to sciview, so the `Volume` doesn't actually know if the mesh should be published or not.

Reply:
I filed #489 for this. I made a new milestone "unscheduled" and put it in there; I hope this is OK. It's nice to have a way to mark issues that have no particular deadline or urgency, but which we would like to keep open. Then they can be easily filtered out of searches.